small-talk/pages/api/auth2.js

```diff
@@ -0,0 +1,30 @@
+import Mongoboi from "@/db/mongo"
+import { headers } from "next/headers"
+import { NextResponse } from "next/server";
+
+export default async function POST(req,res) 
+{
+    let session_id;
+    console.log(req.headers)
+    if (req.headers.authorization) 
+    {
+        session_id = req.headers.authorization
+        console.log("AUTH2: "+ session_id)
+    }
+    else
+    {
+        res.status(200).json({ found: false }); return;
+    }
+
```

Comment:
This makes me realize we currently have the user and pass hard-coded into a const right now